File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Java/Pattern2.java
/*Program 2: Write a Program to Print following Pattern
1
2 3
4 5 6
7 8 9 10
*/

class Patter2{
	
	public static void main(String args[]){
		int num = 1;
		for(int outer = 1 ; outer <=4; outer++ ){

			for(int inner = 1 ; inner <=outer; inner++){
			
				System.out.printf("%d",num++);
			}		
			System.out.printf("\n");
		}
	}	

}	

File: Python Code/Daily_Flash_Code_0/Week 2/23Jan/09-DailyFlash_Solutions/09-DailyFlash_Solutions/23_Jan_Solutions_One/Cpp/pattern2.cpp
/*Program 2: Write a Program to Print following Pattern
1
2 3
4 5 6
7 8 9 10
*/

#include<iostream>


int main(){

	int num =1;
	for(int outer = 1 ; outer <=4; outer++ ){

		for(int inner = 1 ; inner <=outer; inner++){
			
			printf("%d\t",num++);
		}		
		printf("\n");
	}	
}	
